### ISSUE TITLE
In game tutorial after installation 

### ISSUE THREAD

**Issue by l1ving** [kami-blue/client#1509]
This is obviously blocked by #1376, #1283 and #790, but this is something that would be nice to have. 

Some things to add are

- [ ] How to open ClickGUI
- [ ] How to rebind ClickGUI
- [ ] How to change prefix
- [ ] How to change GUI scale
- [ ] How to edit the HUD
- [ ] GUI things such as holding alt for precise setting sliders
